BSDmakefile

```diff
@@ -0,0 +1,5 @@
+default:
```

Comment:
If we do it, it would mean that our documentation would not need to tell people to run 'gmake'. On most systems (linux and mac os x anyway) make is already GNU Make. But on BSD systems, GNU make is called gmake, and make is something else. And, naturally, gmake doesn't even exist in (most?) linux distributions or on mac os X.

So I view it more as a way of simplifying the documentation. This is a makefile that BSD make will use that simply calls 'gmake'.

I think that's a worthwhile trade for 1 more listing entry, but as with many of these things, it's a judgement call.